src/zkaf/apps/Cargo.toml

```diff
@@ -22,18 +13,21 @@ env_logger = { version = "0.10" }
 ethers = { workspace = true }
 log = { workspace = true }
 methods = { workspace = true }
-verity-verifier = { workspace = true }
 risc0-ethereum-contracts = { workspace = true }
 risc0-zkvm = { workspace = true, features = ["client"] }
 tokio = { version = "1.35", features = ["full"] }
-serde = "1.0"
+serde = "1.0.215"
 hex = { workspace = true }
-serde_json = "1.0.107"
+serde_json = "1.0.133"
 sha256 = "1.5.0"
 near-jsonrpc-client = "0.9.0"
 near-crypto = "0.21.1"
 near-primitives = "0.21.1"
 near-chain-configs = "0.21.1"
 near-jsonrpc-primitives = "0.21.1"
-near-contract-standards = "4.1.1"
+indexer = { path = "../../near_indexer" }
 dotenv = { workspace = true }
+near-contract-standards = "5.5.0"
+verity-verifier = { git = "https://github.com/usherlabs/verity-dp", rev = "030110e32d1bccca5f2dd5108c6db1d6dc6694cd", version = "0.0.0", default-features = false, features = [
```

Comment:
Ensure all dependencies share the same version/revision - to prevent errors.
ie. What if the Client uses the Verifier aswell, but the version of Client is different this Verifier.